test-cases/longevity/longevity-harry-2h.yaml

```diff
@@ -0,0 +1,16 @@
+test_duration: 240
+stress_cmd: ["/usr/bin/cassandra-harry"]
+
+n_db_nodes: 6
+n_loaders: 2
+n_monitor_nodes: 1
+
+instance_type_db: 'i3.4xlarge'
```

Comment:
Try to use the minimal machine type, even i3.large.
(unless it's really stressing).